egs/wsj_mix/asr1/run.sh

```diff
@@ -115,17 +115,16 @@ if [ ${stage} -le 0 ]; then
     local/wsj0_2mix_data_prep.sh ${wsj_2mix_wav}/wav16k/max ${wsj_2mix_scripts} \
         ${wsj_full_wav} || exit 1;
 
-    ### Or this part is for WSJ mix
```

Comment:
I think it's better to add some clarification comments about the difference between `local/wsj_2mix_data_prep.sh` and `local/wsj0_2mix_data_prep.sh`